styleguide/_themes/global/scss/grid.scss

```diff
@@ -1,10 +1,20 @@
+$max-row-width: 960px;
+
+@mixin rs-make-row($gutter: $grid-gutter-width) {
+  @include make-row($gutter);
+  margin: 0 auto;
+  max-width: $max-row-width;
+}
```

Comment:
no alternatives huh? that sucks. we will have to make sure this is all good in browerstack. Will this effect any of the general content types that utilize containers and rows now?